packages/devtools/devtools/vite.config.ts

```diff
@@ -26,6 +26,8 @@ export default defineConfig({
       '@dxos/config',
       '@dxos/keys',
       '@dxos/log',
+      '@dxos/mosaic',
+      '@dxos/plexus',
```

Comment:
are these not build with esm versions?

Reply:
It is, but for some reason, testbench utilizing cjs entry point and fails with an error. 
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/36420699/217537269-869370a3-9c66-4a09-b5c0-5634ceed7902.png">

Reply:
huh, interesting

Reply:
I guess the issue is inside Kai, because I copied those from Kai